backend/capellacollab/alembic/versions/ca2346be296b_add_tool_integrations_table.py

```diff
@@ -0,0 +1,61 @@
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Add tool_integrations table
+
+Revision ID: ca2346be296b
+Revises: 0ef0826d54e7
+Create Date: 2022-12-19 17:05:07.009764
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "ca2346be296b"
+down_revision = "0ef0826d54e7"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    t_tool_integrations = op.create_table(
+        "tool_integrations",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("tool_id", sa.Integer(), nullable=True),
+        sa.Column(
+            "t4c", sa.Boolean(), nullable=True, server_default=sa.text("false")
+        ),
+        sa.Column(
+            "pure_variants",
+            sa.Boolean(),
+            nullable=True,
+            server_default=sa.text("false"),
+        ),
```

Comment:
I wanted to keep it a bit more "modular". In addition, we can transform it into Pydantic models without manual transformation (otherwise, we would need to map the fields into a subdict `integrations` to make the json a bit more clean). 
Another advantage: It was easier to implement because I didn't have to touch the existing routes.